### PR TITLE
test: mint/burn member tests

### PR DESCRIPTION
**Pull request by 0xGrapefruit**

### CODE REVIEW

**Review by jtlacci — APPROVED** [review]

File: src/lib/services/transaction-service.ts [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+
+/**
+ * Gets safe info from transaction service.
+ *
+ * @param address - Address of the Safe.
+ * @returns - Safe data.
+ */
+export async function getSafeInfo(address: string): Promise<Safe> {
+  // Transaction-service expects a checksum address, but graphql does not checksum its addresses
+  const result = await axios.get(`${gnosisUrl}/safes/${getAddress(address)}`);
```
Comment (jtlacci) [inline src/lib/services/transaction-service.ts:141]: should we be checksumming the return val?

File: test/token.test.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/* eslint-disable jest/prefer-todo */
+import { ethers } from 'ethers';
+import { init, getPod } from '../src';
```
Comment (jtlacci) [inline test/token.test.ts:3]: remove

File: test/token.test.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+const orcanautAddress = 'orcanauts.pod.xyz';
```
Comment (jtlacci) [inline test/token.test.ts:5]: remove

File: src/lib/services/transaction-service.ts (hunk 1 above)
Comment (0xGrapefruit) [reply src/lib/services/transaction-service.ts:141]: We probably checksum the value down the line, and as long as the final product is checksummed I don't think it's that important.

**Pull request merged**